webpack-aot.config.js

```diff
@@ -8,14 +8,9 @@ webpackConfig.module.rules = [
   loaders.tslint,
   loaders.ts,
   loaders.html,
-  { test: /\.css$/, use: 'raw-loader', include: /node_modules/ },
```

Comment:
yes, we need this to handle 3rd party libs that bring in CSS through `styleUrls`